atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/RetryLimitReachedException.java

```diff
@@ -47,6 +58,7 @@ public String getLogMessage() {
 
     @Override
     public List<Arg<?>> getArgs() {
-        return Collections.singletonList(SafeArg.of("numRetries", numRetries));
+        return ImmutableList.of(
```

Comment:
Could you add tests for this method where we check if we putting the argument correctly?